docs/io-overview.md

```diff
@@ -26,17 +30,17 @@ This diagram illustrates the relationship between source, Pulsar, and sink:
 
 ### Source
 
-> Sources **feed data from external systems into Pulsar**.
+Source connectors **feed data from external systems into Pulsar**.
 
-Common sources include other messaging systems and firehose-style data pipeline APIs.
+Common source connectors include other messaging systems and firehose-style data pipeline APIs.
```

Comment:
```suggestion
Common sources include other messaging systems and firehose-style data pipeline APIs.
```
Here refers to "sources" rather than "connectors"